<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>211</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string>Screen 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page2btn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Screen 2</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(213, 60, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>141</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>Screen 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="screen3btn">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Screen 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="screen1btn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Screen 1</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
